### ISSUE TITLE
Update profile after editing it

### ISSUE THREAD

**Issue by antsarandriamihaja** [danamn/OpenCodeReviewer#69]
I wanted to update my profile and submit the updates but nothing happens when I click on the submit button. 
On refreshing profile page, I get an error: path undefined. (path is: http://checkmycode.ca/dashboard#/profile/antsa)

**Comment by antsarandriamihaja** [comment]
I had to go back to checkmycode.ca and then back into my profile again to see the changes... 
I guess this would be something more related to UX.

**Comment by Marie000** [comment]
This is actually a problem with the routing.
With react-router, we have the history set to 'hashHistory', which creates weird paths with '#'s in it. When we refresh the page, it doesn't always work. 
Normally we would need to switch to 'browserHistory' and it would all be fixed. We would get cleaner paths (in your case : 'http://checkmycode.ca/profile/antsa'), but for some reason, the authentication stops working when I do that. From what I understand, it might be because it reloads the page before the authentication completes. 

So in summary, I am aware of the bug, I'm just not sure yet how to fix it. 
I will close this issue because it is already listed in #55
